Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -161,6 +161,19 @@ bool ScrollAnchoringController::didFindPriorityCandidate(Document& document)
     return false;
 }
 
+static bool absolutePositionedElementOutsideScroller(Element& element, ScrollableArea& scroller)
+{
+    if (auto* renderer = element.renderer()) {
+        if (auto* containingBlock = renderer->containingBlock()) {
```

Comment:
It would be better to use `layer->ancestorLayerIsInContainingBlockChain(scrollableLayer);` which is what we do elsewhere for this check.

Reply:
Updated.